packages/eslint-config-finn-prettier/react.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+    extends: [
+        require.resolve('./prettier.js'),
+        'prettier/react',
```

Comment:
why? `prettier/react` _only_ has rules from `eslint-plugin-react`, of which there are none in the normal prettier config. There is a 0% chance of this override biting us in the ass.